redfish-core/lib/log_services.hpp

```diff
@@ -1107,7 +1113,11 @@ inline void requestRoutesSystemLogServiceCollection(App& app)
     BMCWEB_ROUTE(app, "/redfish/v1/Systems/system/LogServices/")
         .privileges(redfish::privileges::getLogServiceCollection)
         .methods(boost::beast::http::verb::get)(
+#ifdef BMCWEB_ENABLE_REDFISH_DBUS_LOG_ENTRIES
+            [](const crow::Request& req,
```

Comment:
Can you also solve this with a [[maybe_unused]] ?
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/47324/2/include/authorization.hpp